podpac/core/interpolation/interpolation.py

```diff
@@ -28,12 +28,15 @@ def _repr_keys(self):
         return super()._repr_keys + ["interpolation"]
 
     def _eval(self, coordinates, output=None, _selector=None):
-        node = Interpolate(interpolation=self.interpolation)
+        node = Interpolate(interpolation=self.interpolation, source_id=self.hash)
```

Comment:
This was an oversight on my part and fixes a bug. The old line 36 would return from cache with a certain set of settings for ALL Nodes. (whoops).